descriptors/annotators/io/LoadPointCloud.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taeDescription xmlns="http://uima.apache.org/resourceSpecifier">
+  <frameworkImplementation>org.apache.uima.cpp</frameworkImplementation>
+  <primitive>true</primitive>
+  <annotatorImplementationName>rs_loadPointCloud</annotatorImplementationName>
+  <analysisEngineMetaData>
+    <name>LoadPointCloud</name>
+    <description>Load Point clouds from files.</description>
+    <version>1.0</version>
+    <vendor/>
+    <configurationParameters>
+        <configurationParameter>
+            <name>folder</name>
+            <type>String</type>
+            <multiValued>false</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+        <configurationParameter>
+            <name>filenames</name>
+            <type>String</type>
+            <multiValued>true</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+    </configurationParameters>
+    <configurationParameterSettings>
+        <nameValuePair>
+            <name>folder</name>
+            <value>
+                <string>/home/anindex/</string>
```

Comment:
this here should be a path to a data folder. Please rename it as such .e.g. path_to_data